static/bidder-params/improvedigital.json

```diff
@@ -32,13 +28,12 @@
                     "type": "integer"
                 }
             },
-            "required": ["w", "h"],
+            "required": [
+                "w",
+                "h"
+            ],
             "description": "Placement size"
         }
     },
-    "oneOf": [{
-        "required": ["placementId"]
-    }, {
-        "required": ["publisherId", "placementKey"]
-    }]
-}
+    "required": ["placementId"]
```

Comment:
This is a breaking change. Before this change, it's valid to provide a `publisherId` and `placementKey` combination. If publishers are satisfying the requirement this way, this change will break them when only a `placementId` now accepted.

Can you provide a change window?

FYI @bretg @bsardo

Reply:
@jbartek25 Can you advice on this please?

Reply:
Hi @SyntaxNode. Thank you for bringing this up. The change already happened. Improve Digital ad server removed support for placementKey already and our PBS Java adapter schema was updated accordingly long time ago: https://github.com/prebid/prebid-server-java/blob/master/src/main/resources/static/bidder-params/improvedigital.json so this change is safe.

Reply:
Hello @SyntaxNode. Do you need any further clarification/information to resolve your query? Otherwise, it will be really appreciated if you could expedite the approval & merger of this PR.

TIA.

Reply:
Looked back over the documentation history... the 'publisherKey' parameter has been gone for about 3 years. publisherId was added to the docs 2 years ago. So it appears that just the Go adapter had fallen behind.

However, I agree it could be a breaking change to a PBS-Go host company. The publisherId param should not be **required** nor the publisherKey field completley **removed** until a major PBS release prompts host companies to review their stored requests for the newly required parameter.

Reply:
👍 tagged as PBS 3.0 which we are tentatively planning to deliver in the next couple of months.